assets/app/scripts/services/projects.js

```diff
@@ -1,19 +1,30 @@
 'use strict';
 
 angular.module('openshiftConsole')
-  .factory('ProjectsService', [
-    '$location',
-    '$q',
-    '$routeParams',
-    'AuthService',
-    'DataService',
+  .factory('ProjectsService',
     function($location, $q, $routeParams, AuthService, DataService) {
+
+      // eliminates empty Annotations
+      // could potentially move this into a shared Util service 
+      var cleanAnnotations = function(resource) {
+          resource
+            .metadata
+            .annotations = _.reduce(
+                            resource.metadata.annotations,
+                            function(memo, val, key) {
+                              if(val) {
+                                memo[key] = val;
+                              }
+                              return memo;
+                            }, {});
+          return resource;
+      };
```

Comment:
Hm, I'm a little worried about potentially removing other empty annotations we don't know about. I'd suggest only removing display name and description to be safe.